db/migrate/20180715202514_optimize_indices.rb

```diff
@@ -0,0 +1,45 @@
+class OptimizeIndices < ActiveRecord::Migration[5.1]
+  def change
+    remove_index :audits, name: "associated_index"
+    remove_index :audits, name: "auditable_parent_index"
+    remove_index :audits, name: "auditable_index"
+    remove_index :audits, [:user_id, :user_type]
+    add_index :audits, [:associated_type, :associated_id]
+    add_index :audits, [:user_type, :user_id]
+    add_index :audits, [:auditable_type, :auditable_id, :version]
+
+    remove_index :bookmarks, [:owner_id, :owner_type]
+    add_index :bookmarks, [:owner_type, :owner_id]
+
+    remove_index :fact_names, :ancestry
+
+    remove_index :fact_values, :fact_name_id
+
+    remove_index :hosts, :type
+
+    remove_index :lookup_values, name: "index_lookup_values_on_priority" if index_exists?(:lookup_values, :match, name: "index_lookup_values_on_priority")
+
+    remove_index :nics, :type
+
+    remove_index :notification_recipients, :user_id
+
+    remove_index :parameters, name: "index_parameters_on_domain_id_and_type" if index_exists?(:parameters, :type, name: "index_parameters_on_domain_id_and_type")
+    remove_index :parameters, name: "index_parameters_on_hostgroup_id_and_type" if index_exists?(:parameters, :type, name: "index_parameters_on_hostgroup_id_and_type")
```

Comment:
These two may be leftovers from `db/migrate/20100525094200_simplify_parameters.rb` migration on certain dbs